dotnet/Examples/TokenAuth.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.IO;
+using VaultSharp;
+using VaultSharp.V1.AuthMethods;
+using VaultSharp.V1.AuthMethods.Token;
+using VaultSharp.V1.Commons;
+
+// WARNING: The code in this hello-world file is potentially insecure. It is not safe for use in production.
+// This is just a quickstart for trying out the Vault client library for the first time.GetSecretWithToken()
+namespace Examples
+{
+    public class TokenAuthExample
+    {
+        /// <summary>
+        /// Fetches a key-vaule secret (kv-v2) after authenticating to Vault via Token authentication 
+        /// </summary>
+        public string GetSecretWithToken()
+        {
+            /* WARNING: Storing any long-lived token with secret access in an environment variable poses a security risk.
+	           Additionally, root tokens should never be used in production or against Vault installations containing real secrets.
+	           See approle-with-response-wrapping.go for an example of how to use wrapping tokens for greater security. */
```

Comment:
I ended up changing this last sentence to 

`// See the files starting in auth-* for examples of how to securely log in to Vault using various auth methods.`

You probably want to do the same, but with a pattern that matches your names.

Reply:
Got it. I'm realizing I didn't read the end of this comment. I shall update it